lms/djangoapps/branding/__init__.py

```diff
@@ -27,6 +33,14 @@ def get_visible_courses():
 
     filtered_by_org = microsite.get_value('course_org_filter')
 
+    filtered_by_db = TileConfiguration.objects.filter(
+        enabled=True,
+    ).values('course_id').order_by('-change_date')
+
+    if filtered_by_db:
+        filtered_by_db_ids = [course['course_id'] for course in filtered_by_db]
+        filtered_by_db_keys = frozenset([SlashSeparatedCourseKey.from_string(c) for c in filtered_by_db_ids])
+        return [course for course in courses if course.id in filtered_by_db_keys]
```

Comment:
No big deal, I'm not worried about this now. But in general, since Python provides `set` as a first class iterable, we should aim to use `set`s over `list`s wherever we can. 

A list comprehension can be converted to a set comprehension by just changing the square brackets into curly ones.

```python
return {
    course
    for course in courses
    if course.id in filtered_by_db_keys
}
```